app/views/org_admin/templates/history.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <div class="col-md-12">
     <h1>
-      <%= _('Template History') %>
+      <%= templates.first.customization_of.present? ? _('Template Customisation History') : _('Template History') %>
```

Comment:
This title change seems more accurate to me however not sure if this will confuse users.